app/views/conversations/_form.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= form_for :conversation, url: :conversations, html: { class: "" } do |f| %>
```

Comment:
```suggestion
<%= form_for :conversation, url: :conversations, do |f| %>
```

Reply:
done

Reply:
![image](https://user-images.githubusercontent.com/68506896/93962806-371c3400-fd8e-11ea-8596-9775c31f3fc2.png)

Reply:
Ok, sorry for mistake